ssl/ssl_lib.c

```diff
@@ -843,6 +843,18 @@ SSL *ossl_ssl_connection_new_int(SSL_CTX *ctx, const SSL_METHOD *method)
         }
         s->ext.supportedgroups_len = ctx->ext.supportedgroups_len;
     }
+    if (ctx->ext.ctx_ext_keyshares) {
+        s->ext.s_ext_keyshares =
+            OPENSSL_memdup(ctx->ext.ctx_ext_keyshares,
+                           ctx->ext.ctx_ext_keyshares_len
+                           * sizeof(*ctx->ext.ctx_ext_keyshares));
+        if (!s->ext.s_ext_keyshares) {
```

Comment:
Nit (to match style guide)
```suggestion
        if (s->ext.s_ext_keyshares == NULL) {
```